source/kraken/maintenance_worker.cpp

```diff
@@ -73,6 +189,106 @@ void MaintenanceWorker::load_data() {
     LOG4CPLUS_INFO(logger, "Loading database duration: " << duration);
 }
 
+void MaintenanceWorker::create_task_queue() {
+    if (!task_queue_created) {
```

Comment:
we can inverse the condition to avoid excessive indentations

Reply:
good idea :+1: